rpcs3/rpcs3qt/tooltips.h

```diff
@@ -73,14 +73,14 @@ class Tooltips : public QObject
 
 		// debug
 
-		const QString ppu_debug                    = tr("Never use this.");
-		const QString spu_debug                    = tr("Never use this.");
-		const QString set_daz_and_ftz              = tr("Never use this.");
-		const QString accurate_getllar             = tr("Never use this.");
-		const QString accurate_putlluc             = tr("Never use this.");
+		const QString ppu_debug                    = tr("Creates PPU logs.\nOnly useful to developers.\nNever use this.");
+		const QString spu_debug                    = tr("Creates SPU logs.\nOnly useful to developers.\nNever use this.");
+		const QString set_daz_and_ftz              = tr("Sets special MXCSR flags to debug errors in SSE operations.\nOnly used in PPU thread when it's not precise.\nOnly useful to developers.\nNever use this.");
+		const QString accurate_getllar             = tr("Accurately processes SPU MFC_GETLLAR operation.\nNever use this.");
+		const QString accurate_putlluc             = tr("Accurately processes SPU MFC_PUTLLUC operation.\nNever use this.");
```

Comment:
```suggestion
		const QString accurate_putlluc             = tr("Accurately processes SPU MFC_PUTLLUC operation.");
```

Reply:
Same above.

Reply:
Done

Reply:
Maybe it's worth removing this notification from accurate_rsx_access?

Reply:
Same with accurate RSX reservations.

Reply:
Done